src/components/forms/MyEmergencyContactsForm/NewEmergencyContactModal/NewEmergencyContactModal.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Button, Col, Form, Modal } from 'react-bootstrap';
+
+class NewEmergencyContactModal extends React.Component {
+  state = {
+    data: {
+      firstName: '',
+      lastName: '',
+      email: null,
```

Comment:
Why is this not empty string like the others?

Reply:
Because email is not a required field. And I didn't want it to create an empty email entry